rustls/src/crypto/mod.rs

```diff
@@ -574,6 +574,15 @@ pub fn default_fips_provider() -> CryptoProvider {
     aws_lc_rs::default_provider()
 }
 
+/// Non-panicking `let (nonce, ciphertext) = ciphertext.split_at(...)`.
```

Comment:
I wonder if this is better kept in a new `src/polyfill.rs`, with a link to the upstream stabilisation issue? (For this one, https://github.com/rust-lang/rust/issues/119128)

Reply:
Done.